src/GraphErrorHandler.ts

```diff
@@ -58,6 +58,12 @@ export class GraphErrorHandler {
 	 *  }
 	 */
 	private static constructErrorFromResponse(error: any, statusCode: number): GraphError {
+		if (!error.error) {
+			// example - Consider error specified is a string when using type AuthProviderCallback = (error: any, accessToken: string | null) => void;
+			const graphError = new GraphError(statusCode, "Microsoft Graph JavaScript Client SDK came across an error while processing the request");
+			graphError.body = error;
+			return graphError;
+		}
 		error = error.error;
```

Comment:
Can we call the function parameter outerError, and this one innerError or something more descriptive of what these objects look like? `error = error.error` is very confusing.

Reply:
Changed the parameter name passed to the function to graphError